conf/messages.en

```diff
@@ -941,4 +941,8 @@ houseConsignment.index.items.additionalReference.addAdditionalReferenceNumberYes
 
 houseConsignment.index.items.packages.addPackageShippingMarkYesNo.title = Do you want to add a shipping mark?
 houseConsignment.index.items.packages.addPackageShippingMarkYesNo.heading = Do you want to add a shipping mark?
-houseConsignment.index.items.packages.addPackageShippingMarkYesNo.error.required = Select yes if you want to add a shipping mark
\ No newline at end of file
+houseConsignment.index.items.packages.addPackageShippingMarkYesNo.error.required = Select yes if you want to add a shipping mark
+
+houseConsignment.index.items.packages.removePackageTypeYesNo.title = Are you sure you want to remove this package from item {1} in house consignment {0}?
+houseConsignment.index.items.packages.removePackageTypeYesNo.heading = Are you sure you want to remove this package from item {1} in house consignment {0}?
```

Comment:
Any reason we have {1} then {0} and don't just swap the args round?

Reply:
Just following the pattern of specifying the houseConsignment index before the itemIndex @jackheslop96